model/project_ref.go

```diff
@@ -1658,27 +1658,16 @@ func FindOneProjectRefWithCommitQueueByOwnerRepoAndBranch(owner, repo, branch st
 
 // EnableWebhooks returns true if a hook for the given owner/repo exists or was inserted.
 func EnableWebhooks(ctx context.Context, projectRef *ProjectRef) (bool, error) {
```

Comment:
this function name and description is wrong now; seems like we don't neven need this function? Maybe just SetTracksPushEvents or something.